docsets/Spacemacs/docset.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Spacemacs",
+    "version": "0.200",
+    "archive": "spaceemacs.tgz",
```

Comment:
You didn't add this file. You added a file called `Spacemacs.tgz.txt` instead.

Reply:
OK, will rename the file and open a new pull request.